### PR TITLE
feature/add-second-signature-in-trs-create: add second signature

### PR DESCRIPTION
**Pull request by Verbalist**

### CODE REVIEW

File: shared/model/transaction.ts [hunk 1]
```diff
@@ -63,6 +68,7 @@ export interface IAssetSendStake extends IAsset {
 
 export interface IAssetVote extends IAsset {
     votes: Array<string>;
+    type?: VoteType;
```
Comment (axelsheva) [inline shared/model/transaction.ts]: I think it is required field

File: shared/validate/schema/model/transaction/create/vote.ts [hunk 2]
```diff
@@ -47,7 +43,13 @@ export const COMPONENTS_TRS_VOTE = [
                     },
                     secret: {
                         type: 'string',
-                        minLength: 1
+                        minLength: 20,
```
Comment (axelsheva) [inline shared/validate/schema/model/transaction/create/vote.ts]: It is incorrect, secret length is near 100 symbols

File: shared/model/transaction.ts (hunk 1 above)
Comment (Verbalist) [reply shared/model/transaction.ts]: we reuse asset, in future yes it`s will be required

File: shared/validate/schema/model/transaction/create/vote.ts (hunk 2 above)
Comment (Verbalist) [reply shared/validate/schema/model/transaction/create/vote.ts]: need create special validator for secret. It`s 13 words.

**Pull request merged**